src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -3011,7 +3010,7 @@ private BoundExpression BindArgumentExpression(BindingDiagnosticBag diagnostics,
                     Debug.Assert(argument is BoundUnconvertedInterpolatedString or BoundBinaryOperator { IsUnconvertedInterpolatedStringAddition: true });
                     TypeWithAnnotations parameterTypeWithAnnotations = GetCorrespondingParameterTypeWithAnnotations(ref result, parameters, arg);
                     reportUnsafeIfNeeded(methodResult, diagnostics, argument, parameterTypeWithAnnotations);
-                    arguments[arg] = BindInterpolatedStringHandlerInMemberCall(argument, arguments, parameters, ref result, arg, receiverType, receiverEscapeScope, diagnostics);
+                    arguments[arg] = BindInterpolatedStringHandlerInMemberCall(argument, arguments, parameters, ref result, arg, receiver, methodResult.LeastOverriddenMember, diagnostics);
```

Comment:
>LeastOverriddenMember [](http://example.com/codeflow?start=152&length=21)

It is not obvious why we have to pass the ```LeastOverriddenMember``` here. If it is a requirement specific to binding an interpolated string handler, perhaps ```BindInterpolatedStringHandlerInMemberCall``` should do that itself rather than relying on the caller to supply the right value.

Reply:
I'll reduce the symbol to just a bool for whether it requires an instance receiver.